composeApp/src/androidUnitTest/kotlin/org/ooni/testing/CreatePreferenceDataStore.kt

```diff
@@ -0,0 +1,14 @@
+package org.ooni.testing
+
+import android.app.Application
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import androidx.test.core.app.ApplicationProvider
+import org.ooni.probe.di.Dependencies
+
+internal actual fun createPreferenceDataStore(): DataStore<Preferences> {
+    val app = ApplicationProvider.getApplicationContext<Application>()
+    return Dependencies.getDataStore(
+        producePath = { app.filesDir.resolve(Dependencies.Companion.DATA_STORE_FILE_NAME).absolutePath },
```

Comment:
We should have a separate file name for testing, to avoid corrupting the dev build preferences.